packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types.ts

```diff
@@ -57,6 +57,11 @@ export interface IChoiceGroupOptionProps extends IChoiceGroupOption {
    * This value is used to group each ChoiceGroupOption into the same logical ChoiceGroup
    */
   name?: string;
+
+  /**
+   * Value is an alias to key that sets a value on the input. Key is unavailable due to being a React protected prop name.
+   */
+  value?: string;
```

Comment:
It seems like we should consider never allowing `key` to be a prop name across our components since they are accessible via `ev.target`. I wonder if `optionKey` be a clearer name here.

Reply:
This seems like a larger issue that requires an audit of all components. But it does make sense to avoid using specific prop names.

Reply:
I think this PR looks good but just wanted to get your thoughts on naming this prop `optionKey` instead.

Reply:
`optionKey` seems desirable since `value` would make one think that it is the value for the option which is not always true.